### PR TITLE
refactor: write PostWork.test using riteway

### PR DESCRIPTION
**Pull request by krobi64**
[PR Process](https://github.com/poetapp/documentation/blob/master/process/pr-review.md#pr-process) - [PR Review Checklist](https://github.com/poetapp/documentation/blob/master/process/pr-review.md#pr-review-checklist)

### Release
 Semantic release is enabled for this repository. Make sure you follow the right commit message convention. 
 We're using semantic-release's default — [Angular Commit Message Conventions](https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#-git-commit-guidelines).

## Description of Changes

References: #585 

refactor the PostWork.test to use the `riteway` library
since this was the last alsatian test, I had to remove `test:integration` out of the build as alsatian will fail if there are no tests. There is one final PR to clean the rest of this up.

### CODE REVIEW

File: tests/integration/API/PostWork.test.ts [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+      expected: '',
+    })
+
+    await delay(5000)
+
+    const retrievedResponse = await getWorkFromNode(ABraveAndStartlingTruth.id)
+    const retrievedClaim = getClaimWithoutTimestamp(await retrievedResponse.json())
+
+    assert({
+      given: 'POST /works with a signed verifiable claim',
```
Comment (warrenv) [inline tests/integration/API/PostWork.test.ts]: How about this wording?
given: 'An existing work with a signed verifiable claim'

Comment (krobi64) [reply tests/integration/API/PostWork.test.ts]: It's not exactly that. For example, is it an existing claim if it hasn't be sent to node yet? I should remove the signed verifiable claim portion, as this iteration is not following the standard yet.

Comment (krobi64) [reply tests/integration/API/PostWork.test.ts]: oops nm. lol

Comment (krobi64) [reply tests/integration/API/PostWork.test.ts]: How abut `given: a work posted to node`?
